backend/src/test/java/gov/cdc/usds/simplereport/service/DiseaseServiceTest.java

```diff
@@ -86,7 +93,8 @@ void getSupportedDiseasesMap_successful() {
         .containsEntry(_service.fluB().getInternalId(), _service.fluB())
         .containsEntry(_service.rsv().getInternalId(), _service.rsv())
         .containsEntry(_service.hiv().getInternalId(), _service.hiv())
-        .containsEntry(_service.hepatitisC().getInternalId(), _service.hepatitisC());
+        .containsEntry(_service.hepatitisC().getInternalId(), _service.hepatitisC())
+        .containsEntry(_service.gonorrhea().getInternalId(), _service.gonorrhea());
```

Comment:
I know you're following the existing pattern, but I don't really understand why we are creating `DiseaseService.gonorrhea()` and the similar methods for the other newer STIs when they don't have usage outside of tests.
at the least I feel like they really belong in the test file as helper methods..
but also the `DiseaseService.<diseaseName>` methods call `getKnownSupportedDiseasesMap` so I'm not sure if we should really be using them to validate the contents of `supportedDiseasesMap` in this test 🤔

Reply:
@mpbrown Pairing with Merethe right now and we're both wondering if you have any context as to why we have these apparent circular tests for the DiseaseService. Given that you are the eng who has added the most diseases to the app 😄

Reply:
I think I might have [gotten this train-rolling](https://github.com/CDCgov/prime-simplereport/pull/8269/files#diff-b7921f75c3b22b79e9a02ee625c91dfe85910c88097e600a042b6f7d9db8aa02) (my bad 😅) because I thought we needed [this pattern](https://github.com/CDCgov/prime-simplereport/blob/1dc9b1cc598f713ce1a0ec56d3d90c2a536bc75c/backend/src/main/java/gov/cdc/usds/simplereport/service/DiseaseService.java#L52) 😅 . But looks like we were only using this for the [DeviceTypeSync](https://github.com/CDCgov/prime-simplereport/blob/1dc9b1cc598f713ce1a0ec56d3d90c2a536bc75c/backend/src/main/java/gov/cdc/usds/simplereport/service/DeviceTypeSyncService.java#L379) which we don't need to do for the STIs we are adding...